recipes/libassert/all/test_v1_package/conanfile.py

```diff
@@ -13,9 +12,6 @@ def build(self):
         cmake.configure()
         cmake.build()
 
-    def imports(self):
```

Comment:
removing this as this should be handled by the run_environment environment in Conan 1.X with legacy generators